rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -32,11 +32,13 @@
                 </or>
             </when>
             <perform>
-                <hint title="Java APIs for local file system" effort="1" category-id="optional">
-                    <message>Java APIs for local file system identified in the file.
-                    
-                    Local files in containers are not permanent storage and can disappear when an instance of the application terminates. Avoid accessing local filesystems from applications running in OpenShift environments, and consider using a database or cloud-based storage system instead.
+                <hint title="Java APIs for local file system" effort="1" category-id="openshift-potential">
+                    <message>
+                    Java APIs for local file system identified in the file.  
```

Comment:
Message:
In a cloud environment an application never assumes that anything stored on disk will be permanently available because a restart (triggered by code deploy, config change, or the execution environment relocating the process to a different physical location) will usually wipe out all local (e.g., memory and filesystem) state.
The file system of a running container can only be used as a brief, single-transaction cache.  
For example, downloading a large file, operating on it, and storing the results of the operation in the database.